.builder/actions/aws_crt_java_test.py

```diff
@@ -22,20 +23,25 @@ def run(self, env):
 
         endpoint = env.shell.get_secret("unit-test/endpoint")
 
+        # We need to force the alpine profile during test so that the right surefire version gets downloaded
+        additional_profiles = ''
+        if current_host() == 'alpine':
+            additional_profiles = '-P alpine '
+
         with self._write_secret_to_temp_file(env, "unit-test/rootca") as root_ca_file, self._write_secret_to_temp_file(env, "unit-test/certificate") as cert_file, \
         self._write_secret_to_temp_file(env, "unit-test/privatekey") as key_file, self._write_secret_to_temp_file(env, "ecc-test/certificate") as ecc_cert_file, \
         self._write_secret_to_temp_file(env, "ecc-test/privatekey") as ecc_key_file:
 
-            test_command = "mvn -P continuous-integration -B test -DredirectTestOutputToFile=true -DreuseForks=false " \
+            test_command = "mvn -P continuous-integration {} -B test -DredirectTestOutputToFile=true -DreuseForks=false " \
                 "-DrerunFailingTestsCount=5 -Daws.crt.memory.tracing=2 -Daws.crt.debugnative=true -Daws.crt.ci=true " \
-                "-Dendpoint={} -Dcertificate={} -Dprivatekey={} -Drootca={} -Decc_certificate={} -Decc_privatekey={}".format(endpoint, \
+                "-Dendpoint={} -Dcertificate={} -Dprivatekey={} -Drootca={} -Decc_certificate={} -Decc_privatekey={}".format(additional_profiles, endpoint, \
                 cert_file.name, key_file.name, root_ca_file.name, ecc_cert_file.name, ecc_key_file.name)
```

Comment:
trivial: use fstrings instead of .format() (they didn't exist until python 3.6, but the world is python 3.7+ these days)

~"{} -B test".format(additional_profiles)~ -> ` f"{additional_profiles} -B test"`